Launcher.h

```diff
@@ -677,11 +679,10 @@ class Launcher : public PluginHost::IPlugin {
         }
 
     private:
-        string Identifier() const override {
```

Comment:
This is useful to identify in the WorkerPool the name of the executing job, so lets keep it. Right now there is no provisioning to indeed override it, but lets keep this as a non-virtual (remove override, it will be a regular method) and then extend the ThreadPool::WorkerType, using SFINAE to find this method and report it back.